index.d.ts

```diff
@@ -1,25 +1,17 @@
-import { Middleware } from 'redux';
+import { Middleware, MiddlewareAPI, Action } from 'redux';
 import { Observable } from 'rxjs/Observable';
 import { Operator } from 'rxjs/Operator';
 
-export declare function createEpicMiddleware(): Middleware;
-
-// ./node_modules/rxjs/Observable.d.ts
-export declare class ActionsObservable<T> extends Observable<T> {
+export declare class ActionsObservable extends Observable<Action> {
+  constructor(actionsSubject: Observable<Action>);
+  lift(operator: Operator<any, Action>);
+  ofType(...key: any[]);
+}
 
-  // ./node_modules/rxjs/Observable.d.ts specifies an `Observable`'s `source`
-  // property like so: `protected source: Observable<any>;`
-  // since `actionsSubject` is being assigned to an `Observable`'s `source`
-  // property it is being given the type: `Observable<any>`
-  constructor(actionsSubject: Observable<any>);
+export declare interface Epic {
+  (action$: ActionsObservable, store: MiddlewareAPI<any>): Observable<Action>;
+}
 
-  // ./node_modules/rxjs/Observable.d.ts specifies operators as generics like so
-  // `protected operator: Operator<any, T>;`
-  lift(operator: Operator<any, T>);
+export declare function createEpicMiddleware(...epics: Epic[]): Middleware;
```

Comment:
Whoops, missed this comment. You're totally right. Going to correct it for a future release.